src/v2/Utils/Hooks/useAppendStylesheet.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from "react"
+
+interface UseAppendStylesheet {
+  id: string
+  body: string
+}
+
+export const useAppendStylesheet = ({ id, body }: UseAppendStylesheet) => {
+  useEffect(() => {
+    if (document.getElementById(id)) {
+      return
+    }
+
+    const css = document.createElement("style")
+
+    css.id = id
+    css.setAttribute("type", "text/css")
+    css.textContent = body
+
+    document.body.appendChild(css)
+  }, [body, id])
+}
```

Comment:
I think recaptcha might also be appending a stylesheet, so that can be updated as well